pkgs/tools/text/proselint/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchurl, buildPythonApplication, click, future, six }:
+
+buildPythonApplication rec {
+  name = "proselint-${version}";
+  version = "0.8.0";
+
+  src = fetchurl {
+    url = "mirror://pypi/s/proselint/${name}.tar.gz";
```

Comment:
I get a 404 error with this URL. Should this be
mirror://pypi/p/proselint/${name}.tar.gz
instead? (Replace /s/ with /p/)